### ISSUE TITLE
Add markdown to static files list

### ISSUE THREAD

**Issue by scaria** [ai/size-limit#80]
RIght now loading a package with static markdown content makes parsing error as it does not know what loader to use.
adding md to the static list at index.js
```
const STATIC =
  /\.(eot|woff2?|ttf|otf|svg|png|jpe?g|gif|webp|mp4|mp3|ogg|pdf|html|ico|md)$/

``` 
fixed the issue. Is there any reason to not add markdown to the list?

**Comment by ai** [comment]
Good idea. Send PR to save your name in project’s history.

**Comment by scaria** [comment]
Pls review. https://github.com/ai/size-limit/pull/81/files

**Comment by ai** [comment]
Released in 1.0.1
